📝 docs(CONTRIBUTING.md): add contributing guidelines to the project

woop opened the pull request:
🔍 chore(.gitignore): add .idea/ directory to ignore list

woop committed:
📝 docs(CONTRIBUTING.md): add contributing guidelines to the project
🔍 chore(.gitignore): add .idea/ directory to ignore list

woop committed:
📝 docs(CONTRIBUTING.md): update step 3 of development workflow to use 'make dev' command instead of 'pip install -e '.[dev]' -U' command

ShreyaR commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,37 @@
+
+1. Ensure tests pass: `make test`
+2. Format your code: `make autoformat`
+3. Update documentation if needed, located in the `docs` directory.
```
Can you add notes here about how to serve the docs locally? `mkdocs serve`

woop replied:
Done

ShreyaR reviewed (commented):
This is awesome! Super helpful 🙌

Left a quick comment, lgtm otherwise

woop committed:
🔧 chore(.gitignore): add .cache directory to gitignore
📝 docs(CONTRIBUTING.md): update step 3 to include serving docs with mkdocs serve

ShreyaR reviewed (approved):
lgtm!
